Libs/Optimize/ParticleSystem/itkParticleDomain.txx

```diff
@@ -0,0 +1,135 @@
+/*=========================================================================
```

Comment:
Is this a new file you're adding?  itkParticleDomain.txx?

Reply:
That was there from where I branched from Oleks's branch back in the day. Not sure what it does...